packages/keys/download-keys

```diff
@@ -88,3 +88,6 @@ fi
 cp -v "$cacheDir/$keysVersion/"*_pk.bin "$outDir"
 
 check_keys "$outDir"
+
+echo "Finished 'download-keys' script from 'penumbra-zone/keys' package"
+exit 0
```

Comment:
Informational output only, no change to script behavior. I shelved rewriting this script to bail out on errors more readily, in order to focus on shipping the container rewrite. Might circle back to this script on a rainy day, though.